packages/vite/src/node/plugins/modulePreloadPolyfill.ts

```diff
@@ -1,12 +1,12 @@
 import type { ResolvedConfig } from '..'
 import type { Plugin } from '../plugin'
-import { isModernFlag } from './importAnalysisBuild'
+import { isEsmFlag } from './importAnalysisBuild'
 
 export const modulePreloadPolyfillId = 'vite/modulepreload-polyfill'
 const resolvedModulePreloadPolyfillId = '\0' + modulePreloadPolyfillId + '.js'
 
 export function modulePreloadPolyfillPlugin(config: ResolvedConfig): Plugin {
-  let polyfillString: string | undefined
+  let code: string | undefined
```

Comment:
I think the original variable name is fine.

Reply:
What do you think of `polyfillCode`?